### DISCUSSION TITLE
Is this driver supports SSL ?

### DISCUSSION THREAD

**Discussion by gaddehyd** [ing-bank/cassandra-jdbc-wrapper#52] [Q&A]
I am planning to use this driver with SSL options, Where can I pass the key store,trust store information and what is the connection string for it.

**Comment by maximevw** [comment]
Hello @gaddehyd,

Yes, SSL is supported by this driver.

Please refer to this [documentation](https://github.com/ing-bank/cassandra-jdbc-wrapper/wiki/JDBC-driver-and-connection-string#securing-connection-with-tls). When you enable SSL, you have to specify truststore and keystore properties into [standard JSSE properties](https://docs.oracle.com/javase/6/docs/technotes/guides/security/jsse/JSSERefGuide.html#Customization).

**Comment by gaddehyd** [comment]
@maximevw Thanks for sharing the links,  "jdbc:cassandra://${host}:${port}/${keyspace}?eanblessl=true" is this correct url connection string ?

**Reply by maximevw** [reply]
Hello @gaddehyd 

Be careful with the typo in `enablessl`. Also, as the default load balancing policy used by the driver requires to specify the local datacenter, your JDBC URL would probably be something like that: 

`jdbc:cassandra://${host}:${port}/${keyspace}?enablessl=true&localdatacenter=${localdc}`
